Library/Formula/openconnect.rb

```diff
@@ -17,19 +14,18 @@ class Openconnect < Formula
     depends_on "libtool" => :build
   end
 
-  # No longer compiles against OpenSSL 1.0.2 - It chooses the system OpenSSL instead.
-  # http://lists.infradead.org/pipermail/openconnect-devel/2015-February/002757.html
-
   depends_on "pkg-config" => :build
   depends_on "gettext"
-  depends_on "gnutls"
+  depends_on "openssl"
   depends_on "oath-toolkit" => :optional
+  depends_on "p11-kit"
   depends_on "stoken" => :optional
   depends_on "libxml2"
+  depends_on "zlib"
 
   resource "vpnc-script" do
-    url "http://git.infradead.org/users/dwmw2/vpnc-scripts.git/blob_plain/a64e23b1b6602095f73c4ff7fdb34cccf7149fd5:/vpnc-script"
-    sha256 "cc30b74788ca76928f23cc7bc6532425df8ea3701ace1454d38174ca87d4b9c5"
+    url "https://git.infradead.org/users/dwmw2/vpnc-scripts.git/blob_plain/78050150f712d96f81dcade1efe5d16d017e8471:/vpnc-script"
+    sha256 "48cdff979f5f941e2740193c4ee2a90f55549908afcb1fcee64a66f8c0fec05f"
```

Comment:
I wonder if the contents of this are unstable? I'm seeing a `sha256` of `cc30b74788ca76928f23cc7bc6532425df8ea3701ace1454d38174ca87d4b9c5`. Maybe worth mirroring to a gist.

When I make suggestions like these, would you rather I make them by pushing to this PR, or holding off until you have the time to look at it?

Reply:
I hit the issue with the hash too, the `cc30b74788ca76928f23cc7bc6532425df8ea3701ace1454d38174ca87d4b9c5` hash is for the old version of the script which lingers in your ~/Library/Caches/Homebrew, if you remove and re-attempt the build, everything should fall into place. The new url should be stable. I picked the link to the most recent commit, not the floating version in HEAD which would change per commit.

Regarding feedback, I don't mind.

Reply:
Oh of course, good catch! I'll clear it out from the cache before the next build.